websocket/_app.py

```diff
@@ -385,7 +385,7 @@ def check():
         def handleDisconnect():
             if reconnect:
                 _logging.warning("websocket disconnected (retrying in 5 seconds)")
```

Comment:
I’d think you’d want the value of `reconnect` in this log message, rather than leaving it hard-coded at 5?

Reply:
yes indeed! very good call :)